projects/tinkerbell/hook/linuxkit_config.yml

```diff
@@ -0,0 +1,80 @@
+kernel:
```

Comment:
we don't need this anymore since we're using the hook.yaml directly right? it can also be removed from gitignore

Reply:
ah good catch!